code/modules/clothing/chameleon.dm

```diff
@@ -72,7 +72,8 @@
 	name = "Select Chameleon Outfit"
 	button_icon_state = "chameleon_outfit"
 	var/list/outfit_options //By default, this list is shared between all instances. It is not static because if it were, subtypes would not be able to have their own. If you ever want to edit it, copy it first.
-
```

Comment:
I swear you're driving me nuts, move the call to parent on line 155 to the first line of the proc, remove this stupid line here and it wont add the outfit selection button if it's a syndicate variant because the creation of the change item button is the thing that creates this one, if it's a syndicate variant and it doesnt create the change item button, it wont create the change outfit button